### PR TITLE
Added implementation in R

### PR DESCRIPTION
**Pull request by fvbakel**
## Description
This is an implementation in R. Details see README.md

## Contributing requirements

* [X] I read the contribution guidelines in CONTRIBUTING.md.
* [X] I placed my solution in the correct solution folder.
* [X] I added a README.md with the right badge(s).
* [X] I added a Dockerfile that builds and runs my solution.
* [X] I selected `drag-race` as the target branch.
* [X] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeR/solution_1/primes.R [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    start <- as.numeric(format(Sys.time(), "%OS3"))
+    passes <- 0
+    while (TRUE) {
+        passes <- passes + 1
+        sieve <- prime_sieve$new()
+        sieve$init_fields(limit)
+        sieve$run_sieve()
+        now <- as.numeric(format(Sys.time(), "%OS3"))
+        duration <- now - start
+        if (duration > time_limit || passes==1) {
```
Comment (rbergen) [inline PrimeR/solution_1/primes.R]: The `if (... || passes==1)` makes that the loop is only executed once, regardless of the time passed.

**Review by rbergen — COMMENTED** [review]
I prefer your previous naming scheme for solution PRs :)

File: PrimeR/solution_1/primes.R (hunk 1 above)
Comment (fvbakel) [reply PrimeR/solution_1/primes.R]: Ah, well spotted, thanks. I used that during my testing, but forgot to remove on the final checkin.

**Comment by fvbakel** [comment]
I resolved the review remark. I also realized that the number of bits reported was incorrect. I did some additional research and testing and found that in the drag-race conditions 32 bits are used.

File: PrimeR/solution_1/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Implementation in R
+
+![Algorithm](https://img.shields.io/badge/Algorithm-base-green)
+![Faithfulness](https://img.shields.io/badge/Faithful-yes-green)
+![Parallelism](https://img.shields.io/badge/Parallel-no-green)
+![Bit count](https://img.shields.io/badge/Bits-32-green)
```
Comment (rbergen) [inline PrimeR/solution_1/README.md]: If the bit count is not 1, the badge colour to use is -yellowgreen instead of -green.

**Review by rbergen — COMMENTED** [review]
Thank you for the new commit, and correcting the bit count per prime. I did find one small thing related to the latter, see below.

File: PrimeR/solution_1/README.md (hunk 2 above)
Comment (fvbakel) [reply PrimeR/solution_1/README.md]: ok

**Comment by fvbakel** [comment]
> I prefer your previous naming scheme for solution PRs :)

Solved that too, didn't notice that :-)

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**